src/Util/Logger/FriendicaProcessor.php

```diff
@@ -47,13 +47,24 @@ public function __invoke(array $record)
 
 		$i = 1;
 
-		// Skip everything that we shouldn't display
-		while (in_array($trace[$i]['function'], $this->skipFunctions) ||
-			!isset($trace[$i - 1]['file']) ||
-			in_array(basename($trace[$i - 1]['file']), $this->skipFiles)) {
-			$i++;
+		while ($this->isTraceClassOrSkippedFunction($trace, $i)) {
+			if (isset($trace[$i]['class'])) {
```

Comment:
Isn't this whole block of code supposed to be in `isTraceClassOrSkippedFunction` as well?

It should return true to keep the loop going and false to make it stop (instead of the `break` statement).

Reply:
I would vote for this (when I had been under the shower I had the similar idea).

I think that this would improve the readability and reduce the code lines. (Which both is great)

Reply:
Thanks for the context details, now I briefly pictured you naked.

Reply:
I thought that too but I said to myself it is maybe a performance optimization which I don't get..